api/rebac/v1/check.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package api.rebac.v1;
+import "google/api/annotations.proto";
+option go_package = "ciam-rebac/api/rebac/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.rebac.v1";
+
+service Check {
+	rpc CheckPermission (CheckPermissionRequest) returns (CheckPermissionResponse) {
+		option (google.api.http) = {
+			post: "/api/authz/v1/check"
+			body: "*"
+		};
+	};
+}
+
+message CheckPermissionRequest {
+	string subject = 1;
+	string subjectid = 2;
+	string operation = 3;
+	string resourcetype = 4;
+	string resourceid = 5;
+	bool auditlog = 6;
```

Comment:
We can leave this out for now, there is a spike around auditing internally where we'll see if we want to add this here or do something else. At check-time was just a proposal, we're not sure if we'll do that yet.
```suggestion
```

Reply:
@Rajagopalan-Ranganathan We need to update the description/criteria in jira to remove the aduitlog part

Reply:
Agred, JIRA updated with the same